exonum/Cargo.toml

```diff
@@ -41,6 +41,7 @@ tokio-executor = "=0.1.4"
 tokio-io = "=0.1.8"
 tokio-retry = "=0.2.0"
 tokio-threadpool = "=0.1.6"
+tokio-dns-unofficial = "=0.4.0"
```

Comment:
"Unofficial" looks promising. :upside_down_face:

Reply:
Yep. This should have been `tokio-dns-ietf-approved`. While still being pre-1.0 :rofl:

I think it only means that the crate is not maintained by Carl Lerche himself. I don't think that's an issue.